### DISCUSSION TITLE
[Feature] Ability to add photos of a person to their detected faces album

### DISCUSSION THREAD

**Discussion by Ep0chalypse** [immich-app/immich#10748] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be cool to be able to add pictures to detected people from images where the person isn’t detected. 

For example Immich detects my friend Bob and does a good job matching his face in other pics. But sometimes Bob isn’t detected. Or maybe Bob is wearing a motorcycle helmet so his face isn’t detected.  

It would be nice to be able to add photos of Bob to his “person” album that was created when Immich ran face detection. A simple menu option similar to “add to album” menu option, but add to person. 

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#7301
